pkg/quay/quay.go

```diff
@@ -370,3 +378,15 @@ func (c *QuayClient) GetAllRobotAccounts(organization string) ([]RobotAccount, e
 	}
 	return response.Robots, nil
 }
+
+// If robotName is in longform, return shortname
+func handleRobotName(robotName string) (string, error) {
```

Comment:
thinking if we could do (as an extra input validation) a check if:
* the name is empty 
* it contains invalid robot name like "org+"

and those cases could be added to unit tests

Reply:
Maybe use the regexp provided by the quay api with the exception of the longname form?